### ISSUE TITLE
Add the possibility of not using Vista/7 features in versions higher than Windows XP

### ISSUE THREAD

**Issue by OrthodoxWindows** [vxiiduu/NotepadEx#6]
Among these features, I did not vote for any improvements, but rather regressions.

Among these functionalities, I do not see any improvement, but rather regressions. It's all about these three features
- The task dialogs
- The new-style file dialogs
- The new-style windows help

The "do you want to save" prompt is problematic with the classic theme. If I use the original version of Windows XP, I get this :

![notepad_ea8KJ74bnM](https://github.com/vxiiduu/NotepadEx/assets/100134023/605d50cf-2244-4723-b2e8-26f2f4b492b0)

With the new "do you want to save" prompt, I get this :

![npex64(1)_T4Frw9yAbk](https://github.com/vxiiduu/NotepadEx/assets/100134023/f300b667-3391-475a-aae9-75aa8b406e0c)

There is a big problem with the size of the buttons. The ability to use the old version would be great.

Regarding the new-style file dialogs, I don't have a preference, but some users prefer the old style, because the dialog loads faster.

Finally, I find that the new style of Windows help is less efficient than the old one, because it requires an internet connection, whereas the old one did not. I would also greatly appreciate the possibility of restoring the old one.

In any case, thank you very much for this project.

**Comment by vxiiduu** [comment]
There appears to be an issue with your Windows because the task dialogs seem to appear fine when I enable classic theme.

![image](https://github.com/vxiiduu/NotepadEx/assets/73044267/8954e946-16fd-455d-a501-bc69c8cb9b82)

Regardless, I will add either a settings menu or some registry options you can set to get the old style dialog. This will be present in the next release. Thanks for your feedback.

**Comment by OrthodoxWindows** [comment]
Thank you for your reply.

In fact, I'm in Windows 10, all dialogs of this type display badly with the classic theme, it's a known problem. This also happens in Windows 10 notepad. It's just not very annoying because not many programs use this dialog box. Otherwise my system is at 100% DPI.

I actually think that including in the registry the possibility of restoring the old task dialogs is a good idea. The ideal would be to include the possibility of restoring the old windows help in the same way.

**Comment by MiraakThuri** [comment]
Classic theme does not exist in Windows 10 without mods
the buttons display correctly on W7 with classic theme, so i don't think vxiiduu can fix an issue caused by custom theming

tho i do agree a registry switch for old dialogs on post XP could be interesting + pepole can still import the help files from XP and use them

**Comment by OrthodoxWindows** [comment]
> 

Yes, you summed it up pretty well. Indeed, I do not ask for more than the registry key.
Regarding the help file, I'm talking about the link to the file, in the help menu. Importing an offline help file does not change the pointing to the Microsoft site in the help menu.

**Comment by aubymori** [comment]
[This Windhawk mod](https://winclassic.net/thread/1902/classic-task-dialog-fix) can fix task dialog buttons on Windows 10.
